.travis.yml

```diff
@@ -28,7 +28,9 @@ install:
 script:
   - bash rungas.sh
   - grep -L -r --include *.go --exclude-dir vendor -P "Copyright (\d{4}|\(c\)) Microsoft" ./ | tee /dev/stderr | test -z "$(< /dev/stdin)"
-  - go build $(go list ./... | grep -v vendor)
+  - echo -e "travis_fold:start:build.packages\033[33;1mgo build -v $(go list ./... | grep -v vendor)\033[0m"
```

Comment:
???

Reply:
Adding the `-v` switch to build produces a lot of output so I'm trying to fold it in the travis log, see https://github.com/travis-ci/docs-travis-ci-com/issues/949.  Still playing around with it.